docs/UserGuide.md

```diff
@@ -2,15 +2,25 @@
 layout: page
 title: User Guide
 ---
-
-Vax@NUS is a **one stop management app to efficiently track and schedule COVID-19 vaccinations for NUS students.** It is a desktop app **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Vax@NUS can get your appointment management tasks done faster than traditional GUI apps.
-
 * Table of Contents
 {:toc}
 
 :information_source: This icon indicates helpful notes 
 
 --------------------------------------------------------------------------------------------------------------------
+## Introduction
+
+Vax@NUS was conceived at the start of Singapore's COVID-19 vaccination campaign. Our team sought to streamline the 
+vaccine rollout for the University Health Centre(UHC), responsible for administering vaccines to NUS students, when they become eligible to receive the vaccine. 
+Feedback gathered from staff revealed that existing systems were be slow, and difficult to use, potentially impacting the speed of the rollout. 
+
+Therefore, Vax@NUS was developed. It is a **one stop management app to efficiently track and schedule COVID-19 vaccinations for NUS students.** 
+It is a desktop app **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI).
+If you can type fast, Vax@NUS can get your appointment management tasks done faster than traditional GUI apps.
+
+This project was done as part of the module CS2103T Software Engineering at the School of Computing of the National University of Singapore.
```

Comment:
Do we need to include this? Or would it a better idea to omit this?

Reply:
No need to include, but I'm keeping it. Doesn't hurt, gives credit, explains context.

Reply:
I agree. Can leave this part in.

Reply:
It feels a bit out of place to write it in the user guide?  Seems more appropriate to write it in readme.md instead.. People who read our UG are users who wants to learn how to use our application and do we need them to know that this is a CS2103 project in the user guide though?